### DISCUSSION TITLE
Version considered as 'stable'

### DISCUSSION THREAD

**Discussion by BraINstinct0** [valinet/ExplorerPatcher#384] [Q&A] [answered]
I know that the Windows Devs constantly roll out breaking changes, which means that ExplorerPatcher will inevitably be updated frequently.
However I'm thinking of translating it for personal use(but will let others also see it), so if there's/there'll be a version considered stable, I'd like to start from there.
Which, if any, would be one? Or will there ever be one?

**Comment by Anixx** [comment, marked as answer]
The later is the better.

**Reply by BraINstinct0** [reply]
Now it seems like, latest 'release', not prerelease, would work

**Comment by BraINstinct0** [comment]
Forgot to add but this is in regards to #363 .
I figured 'if it can be thoroughly done in one language, maybe it'll make localization easier for other languages'.
Soo well it seems like I'll have to rush before a new release is made...
